auxx/src/HashInstaller.hs

```diff
@@ -0,0 +1,17 @@
+module Main where
+
+import qualified Data.ByteString.Lazy as BSL
+import           Formatting           (sformat, string, (%))
+import           Pos.Crypto           (Hash, SignTag (SignUSVote),
+                                       emptyPassphrase, encToPublic, hash,
+                                       hashHexF, safeSign, unsafeHash,
+                                       withSafeSigner, withSafeSigners)
```

Comment:
Most of those imports are not require, I guess..
Also maintaining them is a pain, so just using qualified names might help.

Reply:
i'm rebuilding with `-Wall` to see what is extra